.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_size = 4
+end_of_line = lf
+
+[*.{c,h,cxx,am,m4}]
+indent_style = tab
```

Comment:
Oh you mean for display.  Yeah sure then width doesn't matter for spaces.
And to check GitHub it's fairly easy, just browse the file as changed by this PR: https://github.com/techee/geany/blob/a3ab9b63f731498a8b70074f8355beb343cfbfd3/src/editor.c#L142  and see it works :)